### PR TITLE
Update team pages

### PR DESCRIPTION
**Pull request by josephsavona**
Updates the team page (on both the current and beta site) to reflect the current core team members.

[Preview](https://beta-reactjs-org-git-new-team-page-fbopensource.vercel.app/learn/meet-the-team)

### CODE REVIEW

**Review by rickhanlonii — APPROVED** [review]
Exciting! 

Mentioned offline, but let's move the employment status to a secondary heading so the sidebar just says the name!

File: beta/src/utils/prepareMDX.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const PREPARE_MDX_CACHE_BREAKER = 2;
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 export function prepareMDX(rawChildren) {
-  const toc = getTableOfContents(rawChildren);
+  const toc = getTableOfContents(rawChildren, /* depth */ 10);
```
Comment (gaearon) [inline beta/src/utils/prepareMDX.js:15]: Bump the cachebreaker above in this file :)

File: beta/src/content/learn/meet-the-team.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Current members of the React team are listed in alphabetical order below.
     Jason likes having large volumes of Amazon packages delivered to the office so that he can build forts. Despite literally walling himself off from his team at times and not understanding how for-of loops work, we appreciate him for the unique qualities he brings to his work. 
 </TeamMember>
 
-<TeamMember name="Joe Savona" permalink="joe-savona" photo="/images/team/joe.jpg" github="josephsavona" twitter="en_JS" title="Engineer at Meta">
+<TeamMember name="Joe Savona" permalink="joe-savona-aaaaaaaabcd" photo="/images/team/joe.jpg" github="josephsavona" twitter="en_JS" title="Engineer at Meta">
```
[the remark this answers is not in the document]
Comment (josephsavona) [reply beta/src/content/learn/meet-the-team.md:28]: dogscience cachebreaker

File: beta/src/utils/prepareMDX.js (hunk 1 above)
Comment (josephsavona) [reply beta/src/utils/prepareMDX.js:15]: TIL thanks!

File: beta/src/content/learn/meet-the-team.md [hunk 3]
```diff
@@ -13,85 +13,73 @@ The React Core team members work full time on the core component APIs, the engin
...
+    Joe was planning to major in math and philosophy but got into computer science after writing physics simulations in Matlab. Prior to React, he worked on Relay, RSocket.js, and the Skip programming language. While he’s not building some sort of reactive system he enjoys running, studying Japanese, and spending time with his family.
+</TeamMember>
 
-### Brian Vaughn {/*brian-vaughn*/}
+<TeamMember name="Josh Story" permalink="josh-story" photo="/images/team/josh.jpg" github="gnoff" twitter="joshcstory" title="Engineer at Vercel">
+    Josh majored in Mathematics and discovered programming while in college. His first professional developer job was to program insurance rate calculations in Microsoft Excel, the paragon of Reactive Programming which must be why he now works on React. In between that time Josh has been an IC, Manager, and Executive at a few startups. outside of work he likes to push his limits with cooking.
+</TeamMember>
 
-![Brian Vaughn](../images/team/bvaughn.jpg)
+<TeamMember name="Laruen Tan" permalink="lauren-tan" photo="/images/team/lauren.jpg" github="poteto" twitter="potetotes" personal="no.lol" title="Engineer at Meta">
```
[the remark this answers is not in the document]
Comment (poteto) [reply beta/src/content/learn/meet-the-team.md]: oh nice i can commit from the ui ☺️ i helped!!

File: beta/src/content/learn/meet-the-team.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ Current members of the React team are listed in alphabetical order below.
     Josh majored in Mathematics and discovered programming while in college. His first professional developer job was to program insurance rate calculations in Microsoft Excel, the paragon of Reactive Programming which must be why he now works on React. In between that time Josh has been an IC, Manager, and Executive at a few startups. outside of work he likes to push his limits with cooking.
 </TeamMember>
 
-<TeamMember name="Laruen Tan" permalink="lauren-tan" photo="/images/team/lauren.jpg" github="poteto" twitter="potetotes" personal="no.lol" title="Engineer at Meta">
+<TeamMember name="Lauren Tan" permalink="lauren-tan" photo="/images/team/lauren.jpg" github="poteto" twitter="potetotes" personal="no.lol" title="Engineer at Meta">
```
Comment (josephsavona) [inline beta/src/content/learn/meet-the-team.md:36]: Oops sorry!

File: beta/src/components/MDX/TeamMember.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    throw new Error(
+      'Expected name, title, permalink, and children for ' + name ??
+        title ??
+        permalink ??
+        'unknown'
+    );
+  }
+  return (
+    <div className="pb-6 sm:pb-10">
+      <div className="flex flex-col sm:flex-row height-auto">
```
Comment (harish-sethuraman) [inline beta/src/components/MDX/TeamMember.tsx:46]:
```suggestion
      <div className="flex flex-col sm:flex-row height-auto items-center">
```
It looks better when image is at the center? What do you think 😅 

| before | after |
| -------| ----- | 
| Screenshot 2022-12-08 at 2 38 03 PM |  Screenshot 2022-12-08 at 2 38 21 PM |

Comment (josephsavona) [reply beta/src/components/MDX/TeamMember.tsx:46]: Thanks but I prefer the current design.

**Pull request merged**